### PR TITLE
Make check-content command compatible with new model

### PR DESCRIPTION
**Pull request by adutra**

### CODE REVIEW

File: servers/quarkus-cli/src/intTest/java/org/projectnessie/quarkus/cli/BaseContentPersistTest.java [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+  private void launch(QuarkusMainLauncher launcher, File outputFile, String... args) {
+    List<String> cmdArgs = new ArrayList<>(Arrays.asList(args));
+
+    if (outputFile != null) {
+      cmdArgs.add("--output");
+      cmdArgs.add(outputFile.getAbsolutePath());
+    }
+
+    result = launcher.launch(cmdArgs.toArray(new String[0]));
+  }
```
Comment (dimas-b) [inline servers/quarkus-cli/src/intTest/java/org/projectnessie/quarkus/cli/BaseContentPersistTest.java:88]: Would it be too much trouble to factor this out into a common class wrt `BaseContentTest`?

File: servers/quarkus-cli/src/intTest/java/org/projectnessie/quarkus/cli/ITCheckContentPersist.java [hunk 2]
```diff
@@ -0,0 +1,216 @@
...
+  }
+
+  @Test
+  public void testEmptyRepo(QuarkusMainLauncher launcher) {
+    launchNoFile(launcher, "check-content");
+    assertThat(result.exitCode()).isEqualTo(0);
+  }
+
+  @Test
+  public void testNonExistingKey(QuarkusMainLauncher launcher) throws Exception {
```
Comment (dimas-b) [inline servers/quarkus-cli/src/intTest/java/org/projectnessie/quarkus/cli/ITCheckContentPersist.java:63]: Could we refactor / share test code with `ITCheckContent`?

**Review by dimas-b — COMMENTED** [review]
`CheckContent` changes LGTM. Could we refactor the test code to reduce duplication?

File: servers/quarkus-cli/src/intTest/java/org/projectnessie/quarkus/cli/BaseContentPersistTest.java (hunk 1 above)
Comment (adutra) [reply servers/quarkus-cli/src/intTest/java/org/projectnessie/quarkus/cli/BaseContentPersistTest.java]: It was a bit involved but I managed to remove all duplications using `@Nested` tests.

**Review by snazy — APPROVED** [review]
LGTM - let @dimas-b take a look as well

**Review by dimas-b — APPROVED** [review]

**Pull request merged**